### DISCUSSION TITLE
conta bloqueada

### DISCUSSION THREAD

**Discussion by bensfull** [vercel/community#4681] [Account Issue]
### Question

eu estou tentando criar um conta com este email  benson-pierre@outlook.com mas esta dizendo conta bloqueada sendo que eu nem criei a conta ainda

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We cannot safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
